src/chain_spec.rs

```diff
@@ -100,17 +120,17 @@ impl Alternative {
 
 /// Flint testnet generator
 pub fn flint_config() -> Result<ChainSpec, String> {
-	ChainSpec::from_json_bytes(&include_bytes!("../res/flint-cc3-spec.json")[..])
+	ChainSpec::from_json_bytes(&include_bytes!("../res/flint-cc3-raw-spec.json")[..])
 }
 
 /// Amber testnet generator
 pub fn amber_config() -> Result<ChainSpec, String> {
-	ChainSpec::from_json_bytes(&include_bytes!("../res/amber-cc2-spec.json")[..])
+	ChainSpec::from_json_bytes(&include_bytes!("../res/amber-cc2-raw-spec.json")[..])
 }
 
 /// Mainnet generator
 pub fn mainnet_config() -> Result<ChainSpec, String> {
-	ChainSpec::from_json_bytes(&include_bytes!("../res/mainnet-spec.json")[..])
+	ChainSpec::from_json_bytes(&include_bytes!("../res/mainnet-raw-spec.json")[..])
```

Comment:
What is this `raw` file? that means the other ones are deprecated? I noticed that for the mainnet one at least the telemetry_endpoints were removed in the raw one compared to the original one.

Reply:
I have explained this in the above comment above raw vs human readable. If still unclear, Lets discuss this over slack